docker/run.sh

```diff
@@ -16,10 +17,10 @@ function inject_config() {
     with_entries(select(.key | startswith("APP_CONFIG_")) | .key |= sub("APP_CONFIG_"; "")) |
     to_entries |
     reduce .[] as $item (
-      {}; setpath($item.key | split("_"); $item.value | fromjson)
+      {}; setpath($item.key | split("_"); $item.value | try fromjson catch $item.value)
     )')"
 
-  >&2 echo "Runtime app config: $config"
+  echo >&2 "Runtime app config: $config"
```

Comment:
Was this needed by some env or just a nit? :p

Reply:
Looks like `shfmt` changed it on me actually. I'll revert back.